micropub.php

```diff
@@ -279,16 +293,20 @@ private static function generate_args() {
     if (isset($_POST['url'])) {
       $args['ID'] = url_to_postid($_POST['url']);
     }
-
-
     if (isset($_POST['published'])) {
-      $args['post_date'] = iso8601_to_datetime($_POST['published']);
-      $args['post_date_gmt'] = get_gmt_from_date($args['post_date']);
+      // if edit of existing post do not set time
```

Comment:
please revert, i think we actually want to allow this. http://indiewebcamp.com/irc/2015-07-19#t1437351414216